perfkitbenchmarker/scripts/netperf_test_scripts/netperf_test.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import json
+import logging
+import subprocess
+import sys
+import gflags as flags
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('num_streams', 1, 'Number of netperf processes to run')
+
+flags.DEFINE_string('netperf_cmd', None,
+                    'netperf command to run')
+
+flags.DEFINE_integer('port_start', None,
+                     'Starting port for netperf command and data ports')
+
+
+def Main(argv=sys.argv):
+  # Parse command-line flags
+  try:
+    argv = FLAGS(argv)
+  except flags.FlagsError as e:
+    logging.error('%s\nUsage: %s ARGS\n%s', e, sys.argv[0], FLAGS)
+    sys.exit(1)
+
+  netperf_cmd = FLAGS.netperf_cmd
+  num_streams = FLAGS.num_streams
+  port_start = FLAGS.port_start
+
+  assert(netperf_cmd)
+  assert(num_streams >= 1)
+  assert(port_start)
+
+  stdouts = [None for _ in range(num_streams)]
```

Comment:
You can also write this as `[None] * num_streams`, although I'm not sure which version I prefer. Up to you.